src/stk/molecular/topology_graphs/cage/vertices.py

```diff
@@ -290,14 +290,6 @@ class UnaligningVertex(_CageVertex):
 
     """
 
-    def __init__(self, vertex):
```

Comment:
Why is this being removed?

Reply:
It was a previous request for handling unaligning vertices properly to fix all the init methods to not take the vertex, but instead to be a normal vertex init method.

Reply:
by previous request, do you mean issue?

Reply:
From previous times we discussed this issue (and the other related issues) we went for wanting a normal init method for all vertices. I figured this change is small and fits here.

Reply:
why wasn't it using the normal init method?

Reply:
No idea - a previous implentation by one of us to use `UnaliningVertex(vertex)` and then call `super.__init__(...)` - which we now just do `UnaliningVertex(...)`

Reply:
I guess it was just a convenience thing?